libs/stats/odc/stats/tasks.py

```diff
@@ -451,7 +506,28 @@ def stream_from_sqs(
             sqs_queue = get_queue(sqs_queue)
 
         for msg in get_messages(sqs_queue, visibility_timeout=visibility_timeout, **kw):
-            # TODO: switch to JSON for SQS message body
             token = SQSWorkToken(msg, visibility_timeout)
-            tidx = parse_task(msg.body)
+            tidx, filedb = parse_sqs(msg.body)
+
+            # avoid the download and update again, how ever, we have to setup an exception to handle the unit test filedb
+            if filedb.split('/')[-1] != 'test_tiles.db':
```

Comment:
this might be cleaner if we check whether the  path is s3 or not

Will this work?

```suggestion
            if filedb[:3] == 's3:'
```

Reply:
I changed it as: 
`if urlparse(filedb).scheme == 's3':`